java/src/jmri/jmrit/operations/locations/schedules/SchedulesTableModel.java

```diff
@@ -139,9 +139,9 @@ public String getColumnName(int col) {
             case MODE_COLUMN:
                 return Bundle.getMessage("ScheduleMode");
             case EDIT_COLUMN:
-                return Bundle.getMessage("Edit");
+                return ""; // no title above Edit column (compare Turnout Table)
             case DELETE_COLUMN:
-                return Bundle.getMessage("Delete");
+                return ""; // no title above Delete column (compare Turnout Table)
```

Comment:
Column state (hidden/shown, width, order) cannot be saved if you remove column names.